### PR TITLE
Allow for custom vault path

### PR DESCRIPTION
**Pull request by 1-alex98**
Variable can be set in faf_path.lua

### CODE REVIEW

File: init_faf.lua [hunk 1]
```diff
@@ -176,6 +169,14 @@ local function load_vault(vault_path)
 	mount_contents(vault_path .. '\\maps', '/maps')
 end
 
+load_vault(InitFileDir .. '\\..\\user\\My Games\\Gas Powered Games\\Supreme Commander Forged Alliance')
+load_vault(SHGetFolderPath('PERSONAL') .. 'My Games\\Gas Powered Games\\Supreme Commander Forged Alliance')
+if custom_vault_path != nil then
```
Comment (speed2CZ) [inline init_faf.lua:174]: ~= in Lua

Comment (speed2CZ) [reply init_faf.lua]: Just trying to maintain the same format across all files we work with

**Comment by axel1200** [comment]
If we should merge that this change would need to manually deployed to the server. Also what is the difference between init.lua and init_faf.lua . Which other init.luas should I change

**Comment by speed2CZ** [comment]
Idk, you tell me :D you're the client guy. Anyway each featured mod has it's own init file.

**Comment by axel1200** [comment]
@speed2CZ they are all based on faf one tho... or at least they should be

**Comment by axel1200** [comment]
I like this... whether it is used or not it will not break stuff right?

**Comment by shalkya** [comment]
Ok so coming back on this one : 
init upload have been fixed. So it's relevant to look into these PR again.

"Also what is the difference between init.lua and init_faf.lua."
init_faf.lua is used in the server to create the init.lua that is used with the game (main mod FAF).
other init like init_fafdevelop.lua are used similarly (to deploy the init.lua) for other featured mods.

Where is the faf_path.lua, you are talking about to set the variable ?

**Comment by 1-alex98** [comment]
faf_path.lua is a file written by the client

**Comment by 1-alex98** [comment]
So I guess init.lua and init_fafdevelop.lua would need to be changed as well

**Comment by 1-alex98** [comment]
In faf_path.lua the client writes the path to FA by setting a variable... The same thing could be done with the vault patht... that was the idea behind this

**Comment by 1-alex98** [comment]
https://github.com/FAForever/downlords-faf-client/blob/f845162a3db52563a0c30019b1042c6e39f4a4dd/src/main/java/com/faforever/client/patch/GameUpdaterImpl.java#L117

**Comment by Garanas** [comment]
Closed in favor of the new PR that can be tested.

**Pull request closed without merging**